WordPressUI/WordPressUI/FancyAlert/FancyButton.swift

```diff
@@ -0,0 +1,196 @@
+import Foundation
+import UIKit
+
+
+// MARK: - FancyButton
+//
+open class FancyButton: UIButton {
+
+    /// Style: Primary + Normal State
+    ///
+    @objc public dynamic var primaryNormalBackgroundColor = Primary.normalBackgroundColor
+    @objc public dynamic var primaryNormalBorderColor = Primary.normalBorderColor
+
+    /// Style: Primary + Highlighted State
+    ///
+    @objc public dynamic var primaryHighlightBackgroundColor = Primary.highlightBackgroundColor
+    @objc public dynamic var primaryHighlightBorderColor = Primary.highlightBorderColor
+
+    /// Style: Secondary
+    ///
+    @objc public dynamic var secondaryNormalBackgroundColor = Secondary.normalBackgroundColor
+    @objc public dynamic var secondaryNormalBorderColor = Secondary.normalBorderColor
+    @objc public dynamic var secondaryHighlightBackgroundColor = Secondary.highlightBackgroundColor
+    @objc public dynamic var secondaryHighlightBorderColor = Secondary.highlightBorderColor
+
+    /// Style: Disabled State
+    ///
+    @objc public dynamic var disabledBackgroundColor = Disabled.backgroundColor
+    @objc public dynamic var disabledBorderColor = Disabled.borderColor
+
+    /// Style: Title!
+    ///
+    @objc public dynamic var titleFont = Title.defaultFont
+    @objc public dynamic var primaryTitleColor = Title.primaryColor
+    @objc public dynamic var secondaryTitleColor = Title.secondaryColor
+    @objc public dynamic var disabledTitleColor = Title.disabledColor
+
+    /// Insets to be applied over the Contents.
+    ///
+    @objc public dynamic var contentInsets = Metrics.contentInsets
+
+    /// Indicates if the current instance should be rendered with the "Primary" Style.
+    ///
+    @IBInspectable var isPrimary: Bool = false {
+        didSet {
+            configureButton()
+        }
+    }
+
+
+    // MARK: - LifeCycle Methods
+
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        configureButton()
```

Comment:
We should probably call this from `init` or somewhere else like `didMoveToWindow`, in case the button isn't created in Interface Builder. Otherwise it'll never be configured.

Reply:
Probably best from `didMoveToWindow` is the easiest way!! Thanks for the heads up!!